src/functions/ripple/ripple.functions.ts

```diff
@@ -96,7 +96,10 @@ export function decodeURI(URI: string): RawVault {
       wdTxId: URI.slice(252, 316) === '0'.repeat(64) ? '' : URI.slice(252, 316),
       btcMintFeeBasisPoints: BigNumber.from(`0x${URI.slice(316, 324)}`),
       btcRedeemFeeBasisPoints: BigNumber.from(`0x${URI.slice(324, 332)}`),
-      btcFeeRecipient: URI.slice(332, 398),
+      btcFeeRecipient: (feeRecipient =>
+        feeRecipient.startsWith('02') || feeRecipient.startsWith('03')
+          ? feeRecipient
+          : feeRecipient.replace(/^0+/, ''))(URI.slice(332, 398)),
```

Comment:
are we sure this is safe? is there a chance that the btc address could start with 02?

Reply:
There is currently no bitcoin address type that start with 0:

https://github.com/citizen010/bitcoin-prefixes-address-list